srcpkgs/clickclack/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'clickclack'
+pkgname=clickclack
+version=0.2.2
```

Comment:
`0.2.3` has now been tagged and includes the fixes from my previously mentioned patch series including the cross-compile patch, so that can be dropped as well after updating :)

Btw it appears `./xbps-src update-check clickclack` fails to find an upstream version currently, so adding an `update` file [as per the XBPS source packages manual](https://github.com/void-linux/void-packages/blob/master/Manual.md#checking-for-new-upstream-releases) with `pattern='\s{12}\K[\d.]+'` in it should fix this (for now at least).